### ISSUE TITLE
Error in COBA network using NEURON backend

### ISSUE THREAD

**Issue by chaoming0625** [NeuralEnsemble/PyNN#722]
Dear all, I run a COBA network, with the code listed in https://neuralensemble.org/docs/PyNN/examples/VAbenchmarks.html

I use NEURON backend (which is istalled through the `pip install neuron`), but I got the following error:

```python

$ python pynn.py  neuron COBA
numprocs=1
Traceback (most recent call last):
  File "pynn.py", line 45, in <module>
    sim, options = get_simulator(
  File "/home/chaoming/softwares/miniconda3/envs/brainpy/lib/python3.8/site-packages/pyNN/utility/__init__.py", line 110, in get_simulator
    sim = import_module("pyNN.%s" % args.simulator)
  File "/home/chaoming/softwares/miniconda3/envs/brainpy/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/chaoming/softwares/miniconda3/envs/brainpy/lib/python3.8/site-packages/pyNN/neuron/__init__.py", line 23, in <module>
    from pyNN.neuron.standardmodels.cells import *
  File "/home/chaoming/softwares/miniconda3/envs/brainpy/lib/python3.8/site-packages/pyNN/neuron/standardmodels/cells.py", line 11, in <module>
    from pyNN.neuron.cells import (StandardIF, SingleCompartmentTraub,
  File "/home/chaoming/softwares/miniconda3/envs/brainpy/lib/python3.8/site-packages/pyNN/neuron/cells.py", line 518, in <module>
    class RandomSpikeSource(hclass(h.NetStimFD)):
  File "/home/chaoming/softwares/miniconda3/envs/brainpy/lib/python3.8/site-packages/neuron/hclass3.py", line 110, in __init_subclass__
    raise TypeError(
TypeError: `RandomSpikeSource` implements `__init__` but misses `__new__`. Class must implement `__new__` and call `super().__new__` with the arguments required by HOC to construct the underlying h.NetStimFD() HOC object.

```

**Comment by apdavison** [comment]
Hi, this error appears with the latest version of NEURON. We are working on fixing it. As a workaround, I suggest you try an earlier version of NEURON (7.x)

**Comment by chaoming0625** [comment]
Dear @apdavison , thanks for your suggestions. I fix this problem.

**Comment by apdavison** [comment]
fixed in #727
